app/lib/mappers/mapNonCoreElements.js

```diff
@@ -0,0 +1,34 @@
+const utils = require('../utils');
+
+function getElements(nonCoreElements) {
+  return nonCoreElements && nonCoreElements.nonCoreElement ?
+    utils.asArray(nonCoreElements.nonCoreElement) : [];
+}
+
+function getElement(nonCoreElements, title) {
+  const serviceDetails = getElements(nonCoreElements).find(el => el.elementTitle === title);
+  return serviceDetails ? serviceDetails.elementText : undefined;
+}
+
+function getServiceDetails(nonCoreElements) {
+  return getElement(nonCoreElements, 'Service details');
+}
+
+function getGeneralNotes(nonCoreElements) {
+  return getElement(nonCoreElements, 'General notes');
+}
+
+function getOpeningTimes(nonCoreElements) {
+  return getElement(nonCoreElements, 'Opening times');
+}
+
+function getVenueType(nonCoreElements) {
+  return getElement(nonCoreElements, 'Venue Type');
+}
+
+module.exports = {
+  getGeneralNotes,
```

Comment:
This is a bit picky but i reckon `get` can be removed from these functions. It reads a bit better from the caller `mapNonCoreElements.serviceDetails > mapNonCoreElements.getServiceDetails`, at least IMO.